slave.mk

```diff
@@ -637,6 +637,7 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 	export sonic_asic_platform="$(patsubst %-$(CONFIGURED_ARCH),%,$(CONFIGURED_PLATFORM))"
 	export enable_organization_extensions="$(ENABLE_ORGANIZATION_EXTENSIONS)"
 	export enable_dhcp_graph_service="$(ENABLE_DHCP_GRAPH_SERVICE)"
+	export sonic_debugging_on="$(SONIC_DEBUGGING_ON)"
```

Comment:
Is there a good reason to separate this from `INSTALL_DEBUG_TOOLS`? Maybe we can combine them both into a `BUILD_DEBUG_IMAGE` flag?

Reply:
There will always be cases where we would want to do something separate for INSTALL_DEBUG_TOOLS compared to SONIC_DEBUGGING_ON. The idea I followed was, INSTALL_DEBUG_TOOLS will install additional debug tools (coredumpctl) installed with in the container. This adds some size to the container. So users may not want to do this bit and do it if they really want to debug using gdb.

SONIC_DEBUGGING_ON, will allow the container to have access to /var/log/journal so that it can find matching core reports.

You are correct we will need both for running the gdb, but wanted to keep them separate to give flexibility to the user.